protos/bottle/fulfillment/v1/line_item.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package bottle.fulfillment.v1;
+
+import "bottle/catalog/v1/product.proto";
+
+message LineItem {
+  string id = 1;
+  int32 quantity = 2;
+
+  bottle.catalog.v1.Product catalog_product = 3;
+
+  bool rush = 6;
+  int64 rush_price = 7;
+
+  int64 individual_cost = 4;
+  int64 individual_price = 5;
```

Comment:
Do we need individual break out like this? We've got the total (in cents) and we've got the quantity. Is it necessary to also have these?

Reply:
I can remove the cost because we don't use it right now, but we don't have a total for each line item right now. You get the total by multiplying the `individual_price` and `quantity`

Reply:
I removed the `individual_cost` field

Reply:
Sounds good. Thanks for clarifying :tada: